test/cases/util/declare.result

```diff
@@ -0,0 +1,4 @@
+DECLARE a, b INT;
+not supported: declare a b int default null
```

Comment:
I am confused by these PRs, so the test still says not supported, I am not sure this is what we want.

Reply:
This is a matrix camp task, and we just need to implement the parser.